internal/test/e2e/gouging_test.go

```diff
@@ -112,3 +112,43 @@ func TestGouging(t *testing.T) {
 		return err
 	})
 }
+
+func TestHostMinVersion(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
+	// create a new test cluster
+	cluster := newTestCluster(t, testClusterOptions{
+		hosts:  int(test.AutopilotConfig.Contracts.Amount),
+		logger: newTestLoggerCustom(zapcore.ErrorLevel),
+	})
+	defer cluster.Shutdown()
+	tt := cluster.tt
+
+	// check number of contracts
```

Comment:
this is redundant too, if you provide `hosts` to `newTestCluster` we wait for contracts to be formed with those hosts